src/resolvers/block.ts

```diff
@@ -89,6 +89,46 @@ const blocksRange: IFieldResolver<any, any> = async (obj, { numberRange, hashRan
 const transactions = (obj, args) =>
   obj.transactions.map(tx => ({ ...tx, from: { address: tx.from }, to: { address: tx.to }, inputData: tx.input }));
 
+interface ITransactionsRolesArgs {
+  from?: string;
+  to?: string;
+}
+
+const transactionsRoles: IFieldResolver<any, any> = async (obj, { from, to }: ITransactionsRolesArgs) => {
```

Comment:
The same comments from #16 apply here. We already have all the transactions inside the `obj` source object, so we don't need to call JSON-RPC to fetch them again.